doc/manual/src/glossary.md

```diff
@@ -168,9 +181,10 @@
 
   A high-level description of software packages and compositions
   thereof. Deploying software using Nix entails writing Nix
-  expressions for your packages. Nix expressions are translated to
-  derivations that are stored in the Nix store. These derivations can
-  then be built.
+  expressions for your packages. Nix expressions specify [derviations],
+  which are [instantiated] and stored in the Nix store.
```

Comment:
```suggestion
  which are [instantiated] into Nix store paths.
```

Reply:
I can't decide which of these two I like better (or is more correct)

Reply:
I don't know which is more correct, but the second is definitely less obvious. To the point that this along with your other suggestion below ("instantiated as Nix store paths") are connections that I myself hadn't fully made until now!

My only concern with this wording is that "Nix store path" is also its own jargon, and they're instantiated to a specific *kind* of Nix store path. Maybe "which are [instantiated] into the Nix store as store derivations"? Not sure if that's too much repetition of the same jargon.

Reply:
That also works for me, less jargon is probably better